dev/monitoring/datarepo-monitoring.yaml

```diff
@@ -0,0 +1,108 @@
+---
+create-secret-manager-secret:
+  enabled: true
+  secrets:
+    - secretName: grafana-google-client-secret
+      vals:
+        - kubeSecretKey: GF_AUTH_GOOGLE_CLIENT_SECRET
+          path: secret/dsde/datarepo/dev/clientid-secret.json
+          vaultKey: oauthClientSecret
+
+backend-config:
+  enabled: true
+
+grafana-gcp-managed-cert:
+  enabled: true
+  fullnameOverride: grafana-gcp-managed-cert
+  domainName: grafana.datarepo-dev.broadinstitute.org
+
+prometheus-gcp-managed-cert:
+  enabled: false
+  fullnameOverride: prometheus-gcp-managed-cert
+  domainName: data-prometheus.datarepo-dev.broadinstitute.org
+
+kube-prometheus-stack:
+  enabled: true
+  prometheusOperator:
+    nodeSelector:
+      cloud.google.com/gke-nodepool: dev-node
+    admissionWebhooks:
+      patch:
+        nodeSelector:
+          cloud.google.com/gke-nodepool: dev-node
+    createCustomResource: false
+  prometheus:
+    serviceAccount:
+      annotations:
+        iam.gke.io/gcp-service-account: sidecar-test-mark@broad-jade-dev.iam.gserviceaccount.com      # workloadID annotation for sidecar to write to stackdriver
+    prometheusSpec:
+      serviceMonitorSelectorNilUsesHelmValues: false
+      serviceMonitorSelector: {}
+      serviceMonitorNamespaceSelector: {}
+      podMonitorSelectorNilUsesHelmValues: false
+      podMonitorSelector: {}
+      podMonitorNamespaceSelector: {}
+      containers:
+        - name: stackdriver-exporter
+          image: gcr.io/stackdriver-prometheus/stackdriver-prometheus-sidecar:0.8.0
+          imagePullPolicy: Always
+          args:
+            - --stackdriver.project-id=broad-jade-dev      # project-id
```

Comment:
Curious: does this take variables?  Seems like it might be easy to mess these value over time and if there was a way to globally define for project, region, cluster, etc. that'd be suuuper useful

Reply:
maybe v2 all the helm file def can have all the projects namespaces etc, meaning building a top level one for all vs just one release